openshift_resources/data_protection_application.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+from ocp_resources.resource import NamespacedResource
+
+LOGGER = logging.getLogger(__name__)
+
+
+class DataProtectionApplication(NamespacedResource):
+    api_group = "oadp.openshift.io"
+
+    def __init__(self):
```

Comment:
please pass please pass "name" as well.
I see NamespacedResource has name already, so maybe add super().__init__

Reply:
exactly, yes
we have a few common params we need to pass up the change.
I'll add them